### DISCUSSION TITLE
Guidelines for Contribution

### DISCUSSION THREAD

**Discussion by iguessitsjeff** [gorules/zen#183] [General]
Opened this PR: https://github.com/gorules/zen/pull/181.

I was curious if this was the right process to contribute to the codebase. I can close the PR if it is not the right process, but would at least like to start a conversation on adding the Python Stub File for more informed type hints when using the Python package.

**Comment by ivanmiletic** [comment]
Hi, we do not have guidelines yet. This is something we will try to define in near future
